spec/lib/danger/ci_sources/support/pull_request_finder_spec.rb

```diff
@@ -103,5 +103,29 @@ def open_pull_requests_info
         expect(result.base).to eq "pr 518 base commit sha1"
       end
     end
+
+    context "specify api endpoint of octokit client" do
+      it "By DANGER_GITHUB_API_HOST" do
+        ENV["DANGER_GITHUB_API_HOST"] = "https://enterprise.artsy.net"
+
+        allow(Octokit::Client).to receive(:new).with(
```

Comment:
yeah I always regard `allow` as not an expectation, this would mean those two tests dont actually have an assertion in them (you could make it pass by removing all the production code from the method). However, I played around a bit with the tests and the way rspec seems to work this is not achievable so `allow` and `expect` is only a wording/intention difference here 👍